README.md

```diff
@@ -49,41 +49,56 @@ $ sha256sum -c --strict SHA256SUM.md
-  -provenance string
-    	path to a provenance file
-  -source string
-    	expected source repository that should have produced the binary, e.g. github.com/some/repo
-  -tag string
-    	[optional] expected tag the binary was compiled from
-  -versioned-tag string
-    	[optional] expected version the binary was compiled from. Uses semantic version to match the tag
+$ go run ./cli/slsa-verifier/ verify-artifact --help
+Verifies SLSA provenance on an artifact blob
+
+Usage:
+  slsa-verifier verify-artifact [flags]
+
+Flags:
+      --branch string            [optional] expected branch the binary was compiled from
+      --builder-id string        EXPERIMENTAL: the unique builder ID who created the provenance
+  -h, --help                     help for verify-artifact
+      --print-provenance         print the verified provenance to std out
+      --provenance-path string   path to a provenance file
+      --source string            expected source repository that should have produced the binary, e.g. github.com/some/repo
+      --tag string               [optional] expected tag the binary was compiled from
+      --versioned-tag string     [optional] expected version the binary was compiled from. Uses semantic version to match the tag
+      --workflow-input map[]     [optional] a workflow input provided by a user at trigger time in the format 'key=value'. (Only for 'workflow_dispatch' events). (default map[])
 ```
 
 ### Example
 
 ```bash
-$ go run ./cli/slsa-verifier -artifact-path ~/Downloads/slsa-verifier-linux-amd64 -provenance ~/Downloads/slsa-verifier-linux-amd64.intoto.jsonl -source github.com/slsa-framework/slsa-verifier -tag v1.3.0
+$ go run ./cli/slsa-verifier -provenance-path ~/Downloads/slsa-verifier-linux-amd64.intoto.jsonl --source github.com/slsa-framework/slsa-verifier --tag v1.3.0 ~/Downloads/slsa-verifier-linux-amd64 
 Verified signature against tlog entry index 3189970 at URL: https://rekor.sigstore.dev/api/v1/log/entries/206071d5ca7a2346e4db4dcb19a648c7f13b4957e655f4382b735894059bd199
 Verified build using builder https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v1.2.0 at commit 5bb13ef508b2b8ded49f9264d7712f1316830d10
 PASSED: Verified SLSA provenance
 ```
 
 The verified in-toto statement may be written to stdout with the `--print-provenance` flag to pipe into policy engines.
 
+### Options Details
+
+The following options are supported for SLSA GitHub builders and generators:
+
+| Option | Description |
+| --- | ----------- |
+| `source` | Expects a source, for e.g. `github.com/org/repo`. |
+| `branch` | Expects a `branch` like `main` or `dev`. Not supported or all GitHub Workflow triggers. |
+| `tag` | Expects a  `tag` like `v0.0.1`. Verifies exact tag used to create the binary. Not supported for all triggers. |
+| `versioned-tag` | Like `tag`, but verifies using semantic versioning. |
+| `workflow-input` | Expects key-value pairs like `key=value` |
```

Comment:
Maybe say it's for build triggered via `workflow_dispatch`, with a link to its doc or an example?